src/seedu/addressbook/AddressBook.java

```diff
@@ -383,12 +401,98 @@ private static String executeCommand(String userInputString) {
             return getUsageInfoForAllCommands();
         case COMMAND_EXIT_WORD:
             executeExitProgramRequest();
+        case COMMAND_SORT_WORD:
+        	return executeSortPerson();
+        case COMMAND_EDIT_WORD:
+        	return executeEditPerson(commandArgs);
         default:
             return getMessageForInvalidCommandInput(commandType, getUsageInfoForAllCommands());
         }
     }
 
-    /**
+    private static String executeEditPerson(String commandArgs) {
```

Comment:
would be good if you can provide method comments for each new method introduced.